src/tito/cli.py

```diff
@@ -348,6 +344,11 @@ def main(self, argv):
         BaseCliModule.main(self, argv)
 
         build_dir = os.path.normpath(os.path.abspath(self.options.output_dir))
+
+        if not os.path.exists(build_dir):
+            print("Creating output directory: %s" % build_dir)
+            os.makedirs(build_dir)
```

Comment:
Moving this from base to the build module looks like it might introduce an error if the build dir doesn't exist for the release command, which also uses self.options.output_dir.